public/app/pipeline-orchestrator-worklet-init.js

```
@@ -0,0 +1,294 @@
+      // 5. Forward SABs to ml-worker (if the PipelineOrchestrator has one)
+      this._forwardSABsToMlWorker();
+
+      // 6. Attach SPECTRAL_FRAME listener so VisualizationEngine gets data
+      this._node.port.addEventListener('message', (ev) => {
+        if (ev.data?.type === 'SPECTRAL_FRAME') {
+          // Re-dispatch as a CustomEvent so VisualizationEngine._onWorkletMessage
+          // can subscribe via the standard port.addEventListener path.
+          // (VisualizationEngine attaches directly in app.js attachDspWorkletToVisuals)
+        }
+      });
+      try { this._node.port.start(); } catch (_) {}
```

ESLint

[error] '_' is defined but never used.

(no-unused-vars)